ecommerce/templates/edx/partials/_student_navbar.html

```diff
@@ -3,7 +3,7 @@
 
 {% block navbar_brand %}
     <h1 class="navbar-brand">
-        <img class="navbar-brand-logo" alt="Harvard Medical School | HMX logo" src="{% static 'images/logo_primary.svg' %}"/>
+        <img class="navbar-brand-logo" alt="{{ platform_name }} logo" src="{% static 'images/default-logo.png' %}"/>
     </h1>
 {% endblock navbar_brand %}
 
```

Comment:
For the lines below this I saw that you included them in the theme. Do we have to include them if we only have changes in the `navbar_brand` block?

Reply:
@UmanShahzad, based on my understanding of ecommerce theming (and Django template language), the template in the theme is picked instead of the default in the ecommerce repository. If the default template is not a full template and just overrides some blocks, all the other overrides which we are not changing, have to be included in the theme template because otherwise they will be missing in the page. I will quickly check this in my devstack and post an update.

Reply:
I also checked the campus ecommerce theme repository and found the same sort of full template overrides as I have done here.

Reply:
One possible way to do this would be to override the templates which includes this template and make it include a different template file which extends the default template with this name. But that again will cause maintaining the whole template code for those overridden templates 😐

Reply:
Ah I see, well that basically means ecommerce doesn't support the same kind of theming as the edx-platform, so yeah we have no choice yet but to either change how theming works to match edx-platform or just do full overrides.